changelogs/fragments/docker_container.yml

```diff
@@ -0,0 +1,11 @@
+major_changes:
+  - "docker_container - no longer uses the Docker SDK for Python. It requires ``requests`` to be installed,
+     and depending on the features used has some more requirements. If the Docker SDK for Python is installed,
+     these requirements are likely met (https://github.com/ansible-collections/community.docker/pull/...)."
+  - "docker_container - the module was completely rewritten from scratch (https://github.com/ansible-collections/community.docker/pull/...)."
+breaking_changes:
+  - "docker_container - ``publish_all_ports`` is no longer ignored in ``comparisons`` (https://github.com/ansible-collections/community.docker/pull/...)."
+  - "docker_container - ``exposed_ports`` is no longer ignored in ``comparisons``. Before, its value was assumed to be identical with the value of ``published_ports`` (https://github.com/ansible-collections/community.docker/pull/...)."
+  - "docker_container - ``log_options`` can no longer be specified when ``log_driver`` is not specified (https://github.com/ansible-collections/community.docker/pull/...)."
+  - "docker_container - ``restart_retries`` can no longer be specified when ``restart_policy`` is not specified (https://github.com/ansible-collections/community.docker/pull/...)."
+  - "docker_container - ``stop_timeout`` is no longer ignored for idempotency if told to be not ignored in ``comparisons``. So far it defaulted to ``ignore`` there, and setting it to ``strict`` had no effect (https://github.com/ansible-collections/community.docker/pull/...)."
```

Comment:
Don't forget to replace these with the actual URL to the PR

Reply:
Oooops, thanks for noticing!

Reply:
Fixed in 89bf3da6c763c7d95378d54ad3c681fba3992cec.